src/Lib/Kros.ToDos.Base/Extensions/ResiliencyHttpClientBuilderExtensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Http;
+using Polly;
+using Polly.Extensions.Http;
+using System;
+using System.Net.Http;
+
+namespace Kros.ToDos.Base.Extensions
+{
+    /// <summary>
+    /// Extensions methods for configuring <see cref="PolicyHttpMessageHandler"/> message handlers as part of
+    /// and <see cref="HttpClient"/> message handler pipeline.
+    /// </summary>
+    public static class ResiliencyHttpClientBuilderExtensions
+    {
+        private const int DefaultRetryCount = 5;
+        private const int DefaultFirstExponentialWaitTimeInMilliseconds = 500;
+
+        private const int DefaultHandledEventsAllowedBeforeBreakingCount = 3;
+        private const int DefaultDurationOfBreakInSeconds = 15;
+        
+        /// <summary>
+        /// Adds default's <see cref="PolicyHttpMessageHandler"/> handlers for resiliency.
+        /// </summary>
+        /// <param name="builder">The <see cref="IHttpClientBuilder"/>.</param>
+        /// <returns>An <see cref="IHttpClientBuilder"/> that can be used to configure the client.</returns>
+        /// <remarks>
+        /// <para>
+        /// See the remarks on <see cref="PolicyHttpMessageHandler"/> for guidance on configuring policies.
+        /// </para>
+        /// </remarks>
+        public static IHttpClientBuilder AddResiliencyDefaultPolicyHandler(this IHttpClientBuilder builder)
+        {
+            if (builder == null)
```

Comment:
```suggestion
            if (builder is null)
```

in this case we use `is`. Is faster and more readable.